src/Loop.php

```diff
@@ -275,6 +275,32 @@ public static function setErrorHandler(callable $callback = null)
     }
 
     /**
+     * Retrieve an associative array of information about the event loop driver.
+     *
+     * The returned array MUST contain the following data describing the driver's
+     * currently registered watchers:
+     *
+     *  [
+     *      "defer"         => ["enabled" => int, "disabled" => int],
+     *      "delay"         => ["enabled" => int, "disabled" => int],
+     *      "repeat"        => ["enabled" => int, "disabled" => int],
+     *      "on_readable"   => ["enabled" => int, "disabled" => int],
+     *      "on_writable"   => ["enabled" => int, "disabled" => int],
+     *      "on_signal"     => ["enabled" => int, "disabled" => int],
+     *      "watchers"      => ["referenced" => int, "unreferenced" => int],
+     *  ];
+     *
+     * Implementations MAY optionally add more information in the array but
+     * at minimum the above key => value format MUST always be provided.
+     *
+     * @return array
+     */
+    public function info()
+    {
+        self::get()->info();
```

Comment:
@bwoebi Changes in other branches are dangerous. I always rebase and force push them.